src/pages/Connect.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <section>
```

Comment:
Could this template be more consistently spaced? It's 2 spaces in some areas and 4 in others.